src/jaegertracing/Tracer.h

```diff
@@ -301,6 +303,12 @@ class Tracer : public opentracing::Tracer,
     int _options;
 };
 
+
+// jaegertracing::SelfRef returns a StartSpanOption pointing to the Span which traceID and spanID should become the new Span's IDs
```

Comment:
```
// jaegertracing::SelfRef returns an opentracing::SpanReference which can be passed to Tracer::StartSpan
// to influence the SpanContext of the newly created span. Specifically, the new span inherits the traceID
// and spanID from the passed SELF reference. It can be used to pass externally generated IDs to the tracer,
// with the purpose of recording spans from data generated elsewhere (e.g. from logs), or by augmenting the
// data of the existing span (Jaeger backend will merge multiple instances of the spans with the same IDs).

Reply:
I suggest copying the description to README as well.

Reply:
Updated with minor changes, added to README as well

Also added:
Must be the lone reference, can be used only for root spans

Please let me know if you don't need this line